http/codegen/openapi/openapi_v2_builder.go

```diff
@@ -159,10 +159,34 @@ func securitySpecFromExpr(root *expr.RootExpr) map[string]*SecurityDefinition {
 					switch s.Kind {
 					case expr.BasicAuthKind:
 						sd.Type = "basic"
+						// Generate scopes to add to description
```

Comment:
Would it make sense to refactor these snippets into a helper function something like:
```go
func addScopeDescription(scopes ..., sd *ServiceData) {
    lines := []string{}
    …
        sd.Description += fmt.Sprintf…
    }
}
```
so that it could be called in all these places?

Reply:
Refactored in a separate function